api/src/DTO/Invitation.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+namespace App\DTO;
+
+use ApiPlatform\Core\Annotation\ApiProperty;
+use ApiPlatform\Core\Annotation\ApiResource;
+
+#[ApiResource(
+    collectionOperations: [],
+    itemOperations: [
+        'find' => [
+            'method' => 'GET',
+            'path' => '/{inviteKey}/find',
+        ],
+    ],
+    routePrefix: '/invitations'
+)]
+class Invitation {
+    #[ApiProperty(readable: false, writable: false, identifier: true, example: 'myInviteKey')]
+    public string $inviteKey;
+    public string $campId;
+    public string $campTitle;
+    public ?string $userDisplayName;
+    public ?bool $userAlreadyInCamp;
+
+    public function __construct(string $inviteKey, string $campId, string $campTitle, ?string $userDisplayName, ?bool $userAlreadyInCamp) {
+        $this->inviteKey = $inviteKey;
+        $this->campId = $campId;
+        $this->campTitle = $campTitle;
+        $this->userDisplayName = $userDisplayName;
+        $this->userAlreadyInCamp = $userAlreadyInCamp;
```

Comment:
`$userAlreadyInCamp` kann aktuell `null` sein wenn der User gar nicht eingeloggt ist. Ist das sinnvoll so oder könnten wir einfach als Default `false` angeben?
```suggestion
        $this->userAlreadyInCamp = $userAlreadyInCamp ?? false;
```

Reply:
Würde ich nicht machen.
Falls der User nicht eingeloggt ist, können wir das nicht bestimmen.
Somit ist domänentechnisch null richtig.
Caching technisch könnte dann sogar das falsche drin stehen.